docs/5-design-patterns/hometask/index.ts

```diff
@@ -0,0 +1,20 @@
+import { Shipment } from "./shipment/shipment";
```

Comment:
Is it ok I can't see any Shipment class declared?

Reply:
fixed